src/test/java/edu/cornell/kfs/module/purap/batch/service/impl/JaggaerGenerateContractPartyCsvServiceImplTest.java

```diff
@@ -187,5 +212,151 @@ static Stream<Arguments> jaggaerContractPartyUploadDtoToStringParameters() {
                 Arguments.of(buildVendor1Dto(),  "taxIdentificationNumber=,")
                 );
     }
+    
+    @ParameterizedTest
+    @MethodSource("builderVendorCSVRowArrayParameters")
+    void testBuilderVendorCSVRowArray(JaggaerContractPartyUploadDto vendorDto, String[] expectedCsvData) {
+
+        String[] actualCsvData = jaggaerGenerateContractPartyCsvServiceImpl.builderVendorCSVRowArray(vendorDto);
+        assertEquals(16, expectedCsvData.length);
```

Comment:
Depending on what changes are made to the reorganization of the header labels, these numeric literals could be replaced by array length or collection size calls. The same goes for the similar situation in the address-row-testing method.

Reply:
OK